src/config.rs

```diff
@@ -0,0 +1,35 @@
+use dotenv::dotenv;
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum ReleaseMode {
+    Dev,
+    Prod
+}
+
+fn default_api_port() -> u16 {
+    4000
+}
+
+fn default_timeout() -> u64 {
+    75
+}
+
+#[derive(Deserialize)]
+pub struct Config {
+    #[serde(default = "default_api_port")]
+    pub api_port: u16,
+    #[serde(default = "default_timeout")]
+    pub timeout: u64,
```

Comment:
Any specific reason why we have this timeout field if it's not being used at the time?

Same thing with the release mode, but that one I know it's going to be implemented later.

I think we should either remove it, or already use in when setting up our rocket server

Reply:
I removed these fields, set them up initially, but ended up not paying attention to the fact that they were unusable.